### PR TITLE
pacific: cmake: pass unparsed args to add_ceph_test() 

### PR DESCRIPTION
**Pull request by tchaikov**
backport of #40496 

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/test/crimson/seastore/CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,12 @@ add_executable(unittest-transaction-manager
   test_block.cc
   test_transaction_manager.cc
   ../gtest_seastar.cc)
+<<<<<<< HEAD
```
Comment (rzarzynski) [inline src/test/crimson/seastore/CMakeLists.txt:5]: Oops.

**Review by rzarzynski — CHANGES_REQUESTED** [review]

File: src/test/crimson/seastore/CMakeLists.txt (hunk 1 above)
Comment (tchaikov) [reply src/test/crimson/seastore/CMakeLists.txt]: @rzarzynski fixed and repushed.

**Review by rzarzynski — APPROVED** [review]

**Review by ronen-fr — APPROVED** [review]

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by tchaikov** [comment]
jenkins test api

**Pull request merged**